src/appdev/ApplicationRouter.js

```diff
@@ -102,7 +102,7 @@ class ApplicationRouter<T> {
     return async (req: Request, res: Response, next: NextFunction) => {
       try {
         const content = await this.content(req);
-        LogUtils.log({ path: this.getPath(), query: req.query, response: content });
+        LogUtils.log({ path: this.getPath(), query: req.query });
```

Comment:
why aren't we logging `content` anymore?

Reply:
it's huge, and it was more important when we were trying to debug errors – i think even though we should be logging any time a client makes a request, logging stuff like live tracking responses isn't important